.github/workflows/autodoc.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Autodoc
+
+on: [push]
+
+jobs:
+  autodoc:
+    name: Autodoc
+    if: github.repository == 'aibasel/downward'
```

Comment:
Should we also only run this on the "main" branch? I think this is best configured above in the "on" part, i.e.,

```
on: 
  push:
    [main]
```
or similar.

Reply:
Would this not have to be something like:
```
on:
  push:
    branches:
      - main
```
?

It is important is that we only run this on a revision that is on main. For example, if we ever push to an old release branch (for a bugfix), we don't want this to overwrite the documentation.

Reply:
Done. I took Maltes code, because that is also how it is written in the documentation.

Reply:
Thanks! Malte's code is the correct solution. I hadn't looked up the real syntax.